test.sh

```
@@ -281,8 +281,9 @@ run_pytest() {
 cmd_unit() { run_pytest tests/ --ignore=tests/e2e/ -v "$@"; }
 cmd_coverage() {
     local target="${1:-coverage.xml}"
+    local source="/coverage/$(basename "$target")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)